images/win/scripts/ImageHelpers/ImageHelpers.psm1

```diff
@@ -29,4 +29,5 @@ Export-ModuleMember -Function @(
     'Test-IsWin19'
     'Test-IsWin16'
     'Choco-Install'
+    'Extract-SevenZip'
```

Comment:
I believe this should be `Extract-7Zip` since `7zip` is an official name of the tool and it is more familiar.

Reply:
Changed to` Extract-7Zip`. Thank you!